recipe/meta.yaml

```diff
@@ -33,13 +33,13 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff  # [py27]
```

Comment:
For some weird reason this fails on Python 3. Not a big deal as `gdal` has tiff support bundled and use that if the external lib is unavailable.